### PR TITLE
#373 - Added tags to layout

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #373 
Added tags to layout

### CODE REVIEW

**Review by genryxy — APPROVED** [review]
@olegmoz thanks, approve
UPD: sorry I missed `@since` tag

File: src/test/java/com/artipie/docker/asto/DefaultLayoutTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import com.artipie.docker.RepoName;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link DefaultLayout}.
+ *
+ * @since 0.9
```
Comment (genryxy) [inline src/test/java/com/artipie/docker/asto/DefaultLayoutTest.java:34]: @olegmoz this tag should be `0.8`

**Review by genryxy — CHANGES_REQUESTED** [review]

**Comment by olegmoz** [comment]
@genryxy thanks, corrected

**Review by genryxy — APPROVED** [review]
@olegmoz thanks, approve

**Pull request merged**